project_event/models/project_task.py

```diff
@@ -207,14 +212,61 @@ class Task(models.Model):
     )
     spectators = fields.Char(
         string='Spectators',
-     )
+        size=10,
+        default='-',
+    )
 
     @api.onchange('spectators')
-    def check_numeric(self):
-        try:
-            int(self.spectators)
-        except Exception as exception:
-            raise ValidationError(_('The Input value should be numeric'))
+    def onchange_spectators(self):
+        if self.spectators:
+            if self.spectators == '-':
+                return
+            try:
+                valid_spectators = int(self.spectators)
+            except Exception:
+                raise ValidationError(_('Spectators value should be numeric'))
+            if valid_spectators >= MIN_SPECTATORS_VALUES_LIMIT and \
+                    valid_spectators < MAX_SPECTATORS_VALUES_LIMIT:
+                self.spectators = valid_spectators
+            else:
+                raise ValidationError(_('Spectators value should be in the range['
+                                        + str(MIN_SPECTATORS_VALUES_LIMIT) +
+                                        '-'
+                                        + str(MAX_SPECTATORS_VALUES_LIMIT) +
+                                        ']'))
+        else:
+            self.spectators = '0'
+
+    real_date_start = fields.Datetime(
+        string='Actual Start Time',
+        default=None,
+    )
+    real_date_end = fields.Datetime(
+        string='Actual End Time',
+        default=None,
+    )
+    total_time = fields.Char(
```

Comment:
@eilst better to change real or actual total time e.g real_total_time or actual_total_time

Reply:
@eilst @rimbendhaou  : My change the variable name in all related files